### PR TITLE
Add transparent CSS class for RoomTile

### PR DESCRIPTION
**Pull request by lukebarnard1**

### CODE REVIEW

File: src/skins/vector/css/themes/_base.scss [hunk 1]
```diff
@@ -104,6 +104,8 @@ $roomtile-name-color: rgba(69, 69, 69, 0.8);
 $roomtile-selected-bg-color: rgba(255, 255, 255, 0.8);
 $roomtile-focused-bg-color: rgba(255, 255, 255, 0.9);
 
+$roomtile-transparent-focused-color: rgba(0, 0, 0, 0.1);
```
Comment (ara4n) [inline src/skins/vector/css/themes/_base.scss]: why are we darkening it for focus? the rest of the UI brightens things for focus...

**Comment by ara4n** [comment]
lgtm other than cosmetic query

File: src/skins/vector/css/themes/_base.scss (hunk 1 above)
Comment (lukebarnard1) [reply src/skins/vector/css/themes/_base.scss]: In the light theme it has a white background to begin with, so we can't go any brighter. Would you propose something else?

Comment (ara4n) [reply src/skins/vector/css/themes/_base.scss]: well, we could lighten the foreground even if the background is already saturated. i'll merge it as is though and see how it looks and tweak as needed then.

**Comment by lukebarnard1** [comment]
Fixes https://github.com/vector-im/riot-web/issues/6210

**Pull request merged**